### DISCUSSION TITLE
Hoe word de main menu geopend?

### DISCUSSION THREAD

**Discussion by DaanKorver** [JustinLung/the-game-of-tones-chapterpage#10] [Questions] [answered]
Wij vroegen ons af hoe je bij de main menu komt en hoe je terug gaat. Op het moment lijkt het erop alsof de paginas niet goed gelinked zijn in het prototype

**Comment by stellaniewenhuis** [comment, marked as answer]
Niet alles is inderdaad klikbaar in het prototype. Je kunt teruggaan naar het main menu door links bovenin op het bolletje met menu te klikken. Verder is het de bedoeling dat je op het hoofdmenu komt als je bent ingelogd.
